package-lock.json

```diff
@@ -40,30 +40,30 @@
       }
     },
     "node_modules/@babel/code-frame": {
-      "version": "7.16.7",
```

Comment:
Let's remove all package-lock modifications from the branch since this is not related to this task.